stdlib/public/Python/Python.swift

```diff
@@ -182,6 +171,19 @@ fileprivate extension PythonConvertible {
   }
 }
 
+//===----------------------------------------------------------------------===//
+// `ConvertibleFromPython` protocol
+//===----------------------------------------------------------------------===//
+
+/// A type that can be initialized from a `PythonObject`.
```

Comment:
```suggestion
/// A type whose values can be initialized from a `PythonObject`.
```

Reply:
Discussed in person; the existing wording is actually okay.